### PR TITLE
Merge gitc484 work

### PR DESCRIPTION
**Pull request by cappelaere**
merged branch master into MergeGitc484Work to prepare for ASTER deploymnet testing

### CODE REVIEW

File: cumulus/tasks/delete-ingest-tracking-data/package.json [hunk 1]
```diff
@@ -0,0 +1,51 @@
+{
+  "name": "delete-ingest-tracking-data",
```
Comment (scisco) [inline cumulus/tasks/delete-ingest-tracking-data/package.json]: could you rename this `@cumulus/delete-ingest-tracking-data`? we have the `@cumulus` in front of all the names to make them publishable to npm

Comment (cappelaere) [reply cumulus/tasks/delete-ingest-tracking-data/package.json]: will do.  Thanks.

File: cumulus/tasks/delete-ingest-tracking-data/index.js [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+   * Main task entry point
+   * @return Any An object/array, etc., that consists of the value of given key in the
+   * message payload
+   */
+  async run() {
+    const message = this.message;
+    const tableName = this.config.ingest_tracking_table;
+    const granuleId = this.config.granule_meta.granuleId;
+    const version = this.config.granule_meta.version;
+    const payload = await message.payload;
```
Comment (yjpa7145) [inline cumulus/tasks/delete-ingest-tracking-data/index.js]: Where is this payload defined?  It's a Promise?

File: cumulus/services/sfn-throttler/index.js [hunk 3]
```diff
@@ -0,0 +1,114 @@
+'use strict';
+
+const aws = require('@cumulus/common/aws');
+const log = require('@cumulus/common/log');
+const _ = require('lodash');
+
+/**
```
[the remark this answers is not in the document]
Comment (scisco) [reply cumulus/services/sfn-throttler/index.js]: @yjpa7145 did you add your eslint verification to the `message-updates` branch or master?

File: cumulus/tasks/delete-ingest-tracking-data/index.js [hunk 4]
```diff
@@ -0,0 +1,53 @@
...
+
+/**
+ * Task that deletes ingest tracking data when ingest completes.
+ * Input payload: An object with the keys given in the output_keys configuration parameter and
+ * possibly others.
+ * Output payload: The value of the keys given in the output_keys configuration parameter.
+ */
+module.exports = class DeleteIngestTrackingData extends Task {
+
+  /**
```
[the remark this answers is not in the document]
Comment (cappelaere) [reply cumulus/tasks/delete-ingest-tracking-data/index.js]: Made changes

File: cumulus/services/sfn-throttler/index.js (hunk 3 above)
[the remark this answers is not in the document]
Comment (yjpa7145) [reply cumulus/services/sfn-throttler/index.js]: It's on both branches.

**Pull request merged**